cli/alterations/alteration.py

```diff
@@ -17,52 +17,54 @@
+    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information[env.ALTERATION_MAX_MESSAGES_TAG])
     nifiConfiguration.create(nameCompose, merge)
-    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information["windowSeconds"])
+    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information[env.ALTERATION_WINDOW_SECONDS_TAG])
 
 
 def createDecode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Encode_Decode.json",information)
     merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Mode", "Decode")
-    if "Encoding" in information:
-        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information["Encoding"])
+    if env.ALTERATION_ENCODING_TAG in information:
+        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information[env.ALTERATION_ENCODING_TAG])
     nifiConfiguration.create(nameCompose, merge)
 
 def createEncode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Encode_Decode.json",information)
-    if "Encoding" in information:
-        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information["Encoding"])
+    if env.ALTERATION_ENCODING_TAG in information:
+        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information[env.ALTERATION_ENCODING_TAG])
     nifiConfiguration.create(nameCompose, merge)
 
 
 
 def createAlteration(nifiConfiguration,allInformation):
-    name=allInformation["name"]
-    for alter in allInformation["alterations"]:
-        if alter["action"]=="Merge":
+    name=allInformation[env.NAME_TAG]
+    for alter in allInformation[env.ALTERATION_TAG]:
+        if alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_MERGE_TAG:
             createMerge(nifiConfiguration,alter,name)
-        elif alter["action"]=="Encode":
+        elif alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_ENCODE_TAG:
             createEncode(nifiConfiguration,alter,name)
-        elif alter["action"]=="Decode":
+        elif alter[env.ALTERATION_ACTION_TAG]==env.ALTERATION_DECODE_TAG:
             createDecode(nifiConfiguration,alter,name)
     connectAlteration(nifiConfiguration,allInformation)
 
 
 def connectAlteration(nifiConfiguration,allInformation):
-    name=allInformation["name"]
-    for index,step in enumerate(allInformation["alterations"]):
+    name=allInformation[env.NAME_TAG]
+    for index,step in enumerate(allInformation[env.ALTERATION_TAG]):
```

Comment:
To be consistent, shouldn't be ALTERATION_ALTERATIONS_TAG?

Reply:
I change the tag "ALTERATION_TAG" to "ALTERATION_ALTERATIONS_TAG"